README.md

```diff
@@ -1,7 +1,13 @@
 # CMR Metadata Review
-
 The CMR Metadata Review tool is used to curate NASA EOSDIS collection and granule level metadata in CMR for correctness, completeness and consistency.
 
+## Security Warning
```

Comment:
I think this should come out since we don't actually see the warning banner unless your apart of the github team.